cantaloupe/interim_files/manifests/gottschalk_manifest_w_lacunae.json

```diff
@@ -209,26 +209,26 @@
           ]
         },
         {
-          "@id": "https://cantus.staging.simssa.ca/iiif/lacuna.jpg",
+          "@id": "https://cantus.simssa.ca/iiif/lacuna.jpg",
           "@type": "sc:Canvas",
           "label": "Lacuna",
           "height": 1800,
           "width": 1400,
           "images": [
             {
-              "@id": "https://cantus.staging.simssa.ca/iiif/lacuna.jpg",
+              "@id": "https://cantus.simssa.ca/iiif/lacuna.jpg",
               "@type": "oa:Annotation",
               "motivation": "sc:painting",
-              "on": "https://cantus.staging.simssa.ca/iiif/lacuna.jpg",
+              "on": "https://cantus.simssa.ca/iiif/lacuna.jpg",
               "resource": {
-                "@id": "https://cantus.staging.simssa.ca/iiif/lacuna.jpg/full/full/0/default.jpg",
+                "@id": "https://cantus.simssa.ca/iiif/lacuna.jpg/full/full/0/default.jpg",
```

Comment:
While not incorrect, I generally avoid having .jpg as a part of the identifier (lacuna.jpg) component, since it can very easily be confused with the format component of the IIIF (default.jpg) to some parsers. It also is confusing since the root of the IIIF Image API URL should be the info.json responds, so  `https://cantus.simssa.ca/iiif/lacuna.jpg` should be a 303 redirect to `https://cantus.simssa.ca/iiif/lacuna.jpg/info.json`. 

Since you don't actually need this to be a URL to an actual image, then you should probably remove this and just have something like `https://cantus.simssa.ca/iiif/lacuna`.